### DISCUSSION TITLE
ADB command to enable Accessibility Service

### DISCUSSION THREAD

**Discussion by Droidphilev** [rustdesk/rustdesk#882] [Q&A]
Hi all,

I would like to use Rustdesk to control a pocradio (an Android device designed to be used as a walkie talkie). The UI is highly customised by the manufacturer and eventhough "Accessibilty Service"works fine there is no UI in settings for it. Can anyone tell met the ADB command to start "Accessibilty Service" for Rustdesk please?

regards,
Phil

**Comment by Heap-Hop** [comment]
https://stackoverflow.com/questions/63066398/i-need-an-adb-command-to-enable-an-android-accesibility-option

```
adb shell settings put secure enabled_accessibility_services com.carriez.flutter_hbb/.InputService
```

**Reply by Droidphilev** [reply]
Thank you! will try this asap
